src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,54 @@
+package qna.domain;
+
+import org.hibernate.annotations.Where;
+
+import javax.persistence.CascadeType;
+import javax.persistence.OneToMany;
+import javax.persistence.OrderBy;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public final class Answers {
+
+    @OneToMany(mappedBy = "question", cascade = CascadeType.ALL)
+    @Where(clause = "deleted = false")
+    @OrderBy("id ASC")
+    private List<Answer> answers = new ArrayList<>();
+
+    public Answers() {
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public List<DeleteHistory> delete(User user) {
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+        for (Answer answer : answers) {
+            deleteHistories.add(answer.delete(user));
+        }
+        return deleteHistories;
```

Comment:
```suggestion
    return answers.stream()
                    .map(answer -> answer.delete(user))
                    .collect(Collectors.toList());
```

스트림을 사용하면 이렇게 만들어 볼 수 있을 것 같아요. :)